common/views/themes/default.ts

```diff
@@ -145,28 +152,17 @@ const GlobalStyle = createGlobalStyle<GlobalStyleProps>`
     )}
   `}
   ${utilityClasses}
-  ${base}
+  ${normalize}
+  ${wellcomeNormalize}
+  ${layout}
+  ${container}
+  ${row}
+  ${inlineFonts}
+  ${fonts}
   ${makeFontSizeClasses()}
   ${makeFontSizeOverrideClasses()}
   ${typography}
   ${grid}
-  ${props =>
```

Comment:
We don't have to override the font here anymore, since we've already decided based on the toggle within the `fonts` declaration file